.github/workflows/dependency-review.yml

```diff
@@ -1,11 +1,8 @@
 name: 'Dependency Review'
-on: 
-  - pull_request
-  - workflow_dispatch
```

Comment:
@aeisenberg We filter the type of event: https://github.com/actions/dependency-review-action/blob/e2f8306beb67a0ad1d5fefc49bf74937e5bf231e/src/main.ts#L10-L14. I think this would prevent you from using `workflow_dispatch` here, is that understanding correct?

Reply:
@aeisenberg I re-added `workflow_dispatch` since it wasn't causing any harm, feel free to experiment with it manually!

Reply:
Thanks. I honestly don't remember if it was working. I'll try it out later and remove if it's broken.